pallets/messages/src/lib.rs

```diff
@@ -174,8 +216,8 @@ pub mod pallet {
 		/// # Returns
 		/// * [DispatchResultWithPostInfo](https://paritytech.github.io/substrate/master/frame_support/dispatch/type.DispatchResultWithPostInfo.html) The return type of a Dispatchable in frame.
 		/// When returned explicitly from a dispatchable function it allows overriding the default PostDispatchInfo returned from a dispatch.
-		#[pallet::weight(T::WeightInfo::add(payload.len() as u32, 1_000))]
-		pub fn add(
+		#[pallet::weight(T::WeightInfo::add_onchain_message(payload.len() as u32, 1_000))]
+		pub fn add_onchain_message(
```

Comment:
Just wanted to remind that this rename might break some client side apps (not sure if we have for frequency yet)

Reply:
good point. should we leave the old method defined and just have it point to this one?